neurodsp/tests/test_filt_filter.py

```diff
@@ -2,11 +2,13 @@
 
 from pytest import raises, warns
 
-from neurodsp.tests.settings import FS
+from neurodsp.tests.settings import FS, EPS_FILT
```

Comment:
now that the tests have moved, the `EPS_FILT` import can be dropped - and the import numpy below.